mmv1/third_party/terraform/tpgresource/labels.go

```diff
@@ -8,18 +8,18 @@ import (
 	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
 )
 
-func FlattenLabels(labels map[string]string, d *schema.ResourceData) map[string]interface{} {
+func SetLabels(labels map[string]string, d *schema.ResourceData, lineage string) error {
```

Comment:
nit: Do you mind commenting the function, to explain the behaviour?

Reply:
Sure. Added doc.